modules/ROOT/partials/configuration/end_container_on_empty_block.adoc

```diff
@@ -1,20 +1,26 @@
 [[end_container_on_empty_block]]
 == `+end_container_on_empty_block+`
 
-This option allows you to to split the current container block element if the enter key is pressed inside an empty inner block element.
+This option specifies how the current container block element is split when the Enter key is pressed inside an empty inner block element. The available options are:
 
-*Type:* `+Boolean+`
+* always split the block;
+* split for specific block elements; or
+* do not split the block at all.
 
-*Default value:* `+false+`
+*Type:* `+Boolean+` or `+String+`
 
-*Possible values:* `+true+`, `+false+`
+*Default value:* `+'blockquote'+`
+
+*Possible values:* `+true+`, `+false+` or comma separated `+String+` of element tag names
+
+NOTE: Prior to {productname} 6.1, this option only accepted `+Boolean+` values.
 
 === Example: Using `+end_container_on_empty_block+`
 
 [source,js]
 ----
 tinymce.init({
   selector: 'textarea',  // change this value according to your HTML
-  end_container_on_empty_block: true
+  end_container_on_empty_block: 'blockquote'
```

Comment:
This change shouldn't have been done as we're supposed to show a different value to the default in the examples.
```suggestion
  end_container_on_empty_block: true
```